backend/clubs/models.py

```diff
@@ -965,6 +965,15 @@ def __str__(self):
         )
 
 
+class Search(models.Model):
+    person = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
```

Comment:
This has to be nullable, what if the user is not logged in?

Reply:
Good point!